marketplace/dev/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM gcr.io/cloud-builders/docker
```

Comment:
This intent was to install docker, so that it's available at run time.

Instead, I'll incorporate the build logic from their image: https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/docker/Dockerfile-17.12.0